pkgs/development/libraries/ogre/default.nix

```diff
@@ -8,11 +8,11 @@
 , nvidia_cg_toolkit }:
 
 stdenv.mkDerivation {
-  name = "ogre-1.9.0";
+  name = "ogre-1.9-hg-20160322";
```

Comment:
If you insist on keeping hg in there, the correct naming format is `ogre-hg-2016-03-22`. See https://nixos.org/nixpkgs/manual/#sec-package-naming